tools/ci/travis-install-dependencies.sh

```diff
@@ -97,3 +98,27 @@ if [ ! -f $CACHE_CCACHE_TOKEN ]; then
     # Create token
     touch $CACHE_CCACHE_TOKEN
 fi
+
+
+# Download and compile pkg-config 0.29.2
```

Comment:
This is the first time we download a dependency ourselves automatically, right? This is a shift from our current approach and I am not sure if it would work on some clusters. What is the motivation for this decision?

Reply:
This installs a newer version of pkg-config in the travis runtime. The version from 14.04 is simply to old.
However, this is now obsolete since #843

Reply:
Oh, for a moment I thought this was in our CMakeLists.txt, sorry.